<launch>
  <!-- Arguments -->
  <arg name="cmd_vel_topic" default="/car1/cmd_vel" />
  <arg name="odom_topic" default="/car1/odom_gazebo" />
  <arg name="move_forward_only" default="true"/>

  <!-- move_base -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS" />
    <rosparam file="$(find multi_robot_scenario)/param/costmap_common_params_burger.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find multi_robot_scenario)/param/costmap_common_params_burger.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find multi_robot_scenario)/param/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find multi_robot_scenario)/param/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find multi_robot_scenario)/param/move_base_params.yaml" command="load" />
    <rosparam file="$(find multi_robot_scenario)/param/dwa_local_planner_params_burger.yaml" command="load" />
    <param name="DWAPlannerROS/debug_level" value="DEBUG"/>
    <remap from="/cmd_vel" to="$(arg cmd_vel_topic)"/>
    <remap from="odom" to="$(arg odom_topic)"/>
    <param name="DWAPlannerROS/min_vel_x" value="0.0" if="$(arg move_forward_only)" />
  </node>
</launch>
